wave_matching/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.3)
 PROJECT(wave_matching)
 
+set( CMAKE_BUILD_TYPE release )
+
 find_package(PCL REQUIRED common registration point_types io)
 find_package(OpenCV REQUIRED)
```

Comment:
Make your CMAKE commands CAPITALS, e.g. `SET(....), FIND_PACKAGE(...)`, whether its lower or upper cmake doesn't care but for consistency lets make it caps, it also stands out so we know what is CMAKE what is a file/target